nodes/vector/vector_angle.py

```diff
@@ -0,0 +1,22 @@
+import bpy
+from ... base_types.node import AnimationNode
+
+class VectorAngleNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_VectorAngleNode"
+    bl_label = "Vector Angle"
+    
+    def create(self):
+        self.inputs.new("an_VectorSocket", "A", "a")
+        self.inputs.new("an_VectorSocket", "B", "b")
+        self.outputs.new("an_FloatSocket", "Angle", "angle")
+        self.outputs.new("an_QuaternionSocket", "Rotation Difference", "quat")
+        
+    def getExecutionCode(self):
```

Comment:
lets use yield here instead of a list of lines. 
the "if not any..." is not needed here